### ISSUE TITLE
Misposition on the log directory

### ISSUE THREAD

**Issue by KannanThuvakaran** [Junjue-Wang/EarthVQA#6]
Should be like this:
EarthVQA
├── Train
│   ├── images_png
│   ├── masks_png_v2
├── Val
│   ├── images_png
│   ├── masks_png_v2
├── Test
│   ├── images_png
├── Train_QA.json
├── Val_QA.json
├── Test_QA.json
├── log
     ├── sfpnr50.pth
     ├── soba.pth

**Comment by Junjue-Wang** [comment]
Yes, now I have updated the data dir, i.e., 'masks_png_v2->masks_png'

**Comment by KannanThuvakaran** [comment]
> Yes, now I have updated the data directory, i.e., 'masks_png_v2->masks_png'

What I meant was is the log directory meant to be outside EarthVQA? For me, it worked when moving log dir inside EarthVQA. So the directory tree needs to update if that is the case.

**Comment by Junjue-Wang** [comment]
OKay, I see. Thanks for pointing out this.
